ModularTegustation/tegu_items/extraction/lock.dm

```diff
@@ -8,6 +8,11 @@
 	howtouse = "This tool can only be used on a containment cell that has reached 100% understanding. This does NOT affect Qliphoth Counter."
 	itemname = "Lock"
 
+/obj/item/extraction/lock/examine(mob/user)
+	. = ..()
+	if (GetFacilityUpgradeValue(UPGRADE_EXTRACTION_1))
+		. = "Use on a work console to raise the qliphoth suppression field of the abnormality cell, slowing down work but increases work chance."
```

Comment:
anything adding onto examine() should be encased in some type of span so it can be sorted by chat filters
```suggestion
		. = span_notice("Use on a work console to raise the qliphoth suppression field of the abnormality cell, slowing down work but increases work chance.")
```